### PR TITLE
Applied EsLint rules for the component

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Description
Applied EsLint rules for the component

## Backward compatibility

This change is fully backward compatible.

## Notes
Using [this](https://www.npmjs.com/package/eslint-config-fliplet) EsLint config

### CODE REVIEW

File: js/libs.js [hunk 1]
```diff
@@ -287,29 +287,31 @@ Fliplet.Registry.set('comflipletanalytics-report:1.0:core', function(element, da
...
-          case 'last-6-months':
-            momentTime = moment(this.x).subtract(6, 'months');
-            break;
-          case 'last-12-months':
-            momentTime = moment(this.x).subtract(12, 'months');
-            break;
-          case 'custom-dates':
-            momentTime = moment(this.x).subtract(timeDeltaInMillisecs);
-            break;
+        case 'last-24-hours':
```
Comment (squallstar) [inline js/libs.js]: is this in our rules? I believe `case` should be indented, please fix it as required

File: js/libs.js [hunk 2]
```diff
@@ -380,7 +382,7 @@ Fliplet.Registry.set('comflipletanalytics-report:1.0:core', function(element, da
   }
 
   function attachEventListeners() {
-
+    // eslint-disable-next-line spaced-comment
```
Comment (squallstar) [inline js/libs.js]: why? can't you just fix the source code?

File: js/libs.js (hunk 1 above)
Comment (YaroslavOvdii) [reply js/libs.js]: @squallstar 
Yes, it an EsLint indent rule which is suggesting I fix it like this.

Code before the fix:
image

Should we ignore errors like this?

Comment (squallstar) [reply js/libs.js]: @YaroslavOvdii I have corrected the `indent` rule to allow 2 spaces for the case, see https://github.com/Fliplet/eslint-config-client/commit/171629ef4e210cd11f03db2c5b857dde8e675c2c

Update the `Fliplet/eslint-config-client` repo via npm (version 0.7) to get the update.

cc @tonytlwu @hcarneiro @MaksymShokin

File: js/libs.js (hunk 2 above)
Comment (YaroslavOvdii) [reply js/libs.js]: @squallstar 
Fixed the source code as lint recommended

File: js/libs.js (hunk 1 above)
Comment (YaroslavOvdii) [reply js/libs.js]: @squallstar 
Updated lint config and applied new indent rules.

**Pull request merged**